### PR TITLE
[Feature/isxcsp 91] BreadCrumb

### PR DESCRIPTION
**Pull request by leroj3**
Added BreadCrumb
Note: It's just the Layout. It constains Dummy Data.

### CODE REVIEW

**Comment by leroj3** [comment]
Looks like this now. 
Capture

**Comment by leroj3** [comment]
Check the last piece of code, I have added functionality to the breadcrumb.

**Comment by leroj3** [comment]
I would like to talk to you about the routing structure when you're available.
Right now it looks like this:
nlnl

**Comment by leroj3** [comment]
I don't know why the Sagas worker only appears once... I'm still trying to troubleshoot this.

I added the breadcrumb to the other pages.

Sagaworker
code

File: src/pages/get-help-detail/GetHelpDetail.jsx [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+
+
+const GetHelpDetail = ({location}) => {
+
+    console.log(location.pathname)
 
-const GetHelpDetail = () => {
     return (<Layout>
+        <BCrumb >
+            <Hyperlink url="http://localhost:3000/nl/nl/">
```
Comment (yawsoningka) [inline src/pages/get-help-detail/GetHelpDetail.jsx:17]: Dynamically build the base URL an place it here :
https://stackoverflow.com/questions/1034621/get-the-current-url-with-javascript

File: src/pages/get-help/GetHelp.jsx [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
 import React from "react";
 import Layout from "../../components/layout/Layout"
+import BCrumb from "../../components/Bcrumb/BCrumb";
+import Hyperlink from "@ingka/hyperlink";
+
+// Routes
+import { ROUTE_BASE } from "../../constants/routes";
+
+const GetHelp = ({location}) => {
+    console.log(location.pathname)
```
Comment (yawsoningka) [inline src/pages/get-help/GetHelp.jsx:11]: Remove all console logs

File: src/pages/get-help/GetHelp.jsx [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
 
-const GetHelp = () => {
     return (<Layout>
+        
+        <BCrumb >
+            <Hyperlink url="http://localhost:3000/nl/nl/">
```
Comment (yawsoningka) [inline src/pages/get-help/GetHelp.jsx:16]: Same here

File: src/pages/home/Home.jsx [hunk 4]
```diff
@@ -2,12 +2,23 @@
...
-const Home = () => {
-    return  (<Layout>
-         <Headline />
+const Home = ({location}) => {
+    console.log(location.pathname)
 
-        </Layout >);
+    return (<Layout>
+        <BCrumb >
+            <Hyperlink url="http://localhost:3000/nl/nl/">
```
Comment (yawsoningka) [inline src/pages/home/Home.jsx:13]: Same here

File: src/store/sagas/bcrumb.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import {takeEvery} from "redux-saga/effects";
+import {SET_CURRENT_URL} from "../actions/bcrumb"
+
+
+export function* handleSetURL() {
+  yield console.log("Saga worker")
```
Comment (yawsoningka) [inline src/store/sagas/bcrumb.js:6]: Clean this up

**Review by yawsoningka — COMMENTED** [review]
Left some comments

**Comment by yawsoningka** [comment]
> I don't know why the Sagas worker only appears once... I'm still trying to troubleshoot this.
> 
> I added the breadcrumb to the other pages.
> 
> Sagaworker
> 
> code

Lets review this later

File: src/pages/get-help/GetHelp.jsx (hunk 2 above)
Comment (leroj3) [reply src/pages/get-help/GetHelp.jsx:11]: done

**Comment by leroj3** [comment]
Hi Emmanuel.
See picture below. Now the state of the URL is changing when I click on the crumbs. I did this with Saga, and created a reducer.
I added this piece of code   <Hyperlink url={`${window.location.hostname}/nl/nl`}>
Not sure if that's what you were referring to when you gave me the stackOverflow link. It seems to not be working since the URL repeats itself, but maybe if it's an actual URL and not localhost that problem would not happen.
Now that the state of the URL is changing to the current URL, how would you like me to implement that in the URL? through UseSelectors?

states

**Comment by leroj3** [comment]
Deprecated.

**Pull request closed without merging**